Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "fluent-play"
+version = "0.1.0"
+authors = ["Staś Małolepszy <stas@mozilla.com>"]
+edition = "2018"
+
+[[bin]]
+name = "fluent-play"
+path = "server/main.rs"
+
+[dependencies]
+corsware = "*"
+hubcaps = "*"
+hyper = "*"
+hyper-tls = "*"
+iron = "*"
+router = "*"
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "*"
+tokio = "*"
```

Comment:
That looks very scary. Are you sure that you're infinitely compatible with all versions of all of those libraries? :)

Reply:
Ha, good point. I used `*` when I was developing, and forgot to change them to pinned versions. Thanks!